index.js

```diff
@@ -1016,41 +1020,63 @@ const createFocusTrap = function (elements, userOptions) {
       return this;
     },
 
+    _isManuallyPaused() {
+      return state.manuallyPaused;
+    },
+
+    _setPausedState(paused, options) {
```

Comment:
I was thinking we'd also make these non-enumerable on the trap object since we're wanting them to be internal methods.

```javascript
Object.defineProperty(trap, '_isManuallyPaused', {
  value() {
    return state.manuallyPaused;
  }
});

// and same for _setPausedState()
```

Reply:
Done